sdk/build.gradle

```diff
@@ -24,9 +25,24 @@ dependencies {
     signature "net.sf.androidscents.signature:android-api-level-24:7.0_r2@signature"
 }
 
+sourceSets {
+    main {
+        output.dir("build/generated/properties", builtBy: 'generateVersionResource')
+    }
+}
+
 animalsniffer {
     // Don't check sourceSets.jmh and sourceSets.test
     sourceSets = [
             sourceSets.main
     ]
 }
+
+task generateVersionResource {
+    doLast {
+        def folder = file(propertiesDir)
+        folder.mkdirs()
+        def propertiesFile = new File(folder.getAbsolutePath(), "version.properties")
```

Comment:
With such a generic filename, it seems like there's a possibility this could collide with other projects version file. What do you think about making this be something super clear like "opentelemetry-sdk-version.properties" ?

Reply:
oh, never mind. I see that the folder will be included in here, as well. It's all good!